src/components/ui/notifications/Notifications.vue

```diff
@@ -162,3 +162,41 @@ export default {
   },
 }
 </script>
+
+<style lang="scss">
+.toasted-container {
```

Comment:
Ideally we do not want any global styles to be in presentation.

Imagine user employs toasted in some other place, then deletes that page.